browser/brave_profile_prefs.cc

```diff
@@ -402,6 +403,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
 #endif
 
   de_amp::RegisterProfilePrefs(registry);
+  debounce::RegisterProfilePrefs(registry);
```

Comment:
RegisterProfilePrefs is normally a static method on the service so DebounceService::RegisterProfilePrefs